aimmo-game/tests/functional/test_map_movements.py

```diff
@@ -0,0 +1,195 @@
+from unittest import TestCase
+
+from mock_world import MockWorld
+from simulation.location import Location
+from simulation import map_generator
+from tests.test_simulation.dummy_avatar import MoveEastDummy, MoveWestDummy, MoveNorthDummy, MoveSouthDummy, WaitDummy
+
+
+class TestMapMovements(TestCase):
+
+    SETTINGS = {
```

Comment:
somehow inconsistent, in test_action.py, the settings are not in a SETTINGS dictionary. Maybe open an issue for this later?

Reply:
There are no settings in `test_action.py`? If you mean the constants at the top such as `ORIGIN` or `SOUTH_OF_ORIGIN` then I personally think it's a different case scenario;

`SETTINGS` in `MockWorld` needs to be supplied as a whole dictionary and is used to update the existing dictionary in the World, these values are just constants used on their own.

I can put them into a dictionary and access them individually from there (maybe from an issue); as you wish ;)